src/main/msp/msp.c

```diff
@@ -3252,21 +3254,23 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         if (sbufBytesRemaining(src) >= 1) {
             rxConfigMutable()->fpvCamAngleDegrees = sbufReadU8(src);
         }
-        if (sbufBytesRemaining(src) >= 6) {
+         if (sbufBytesRemaining(src) >= 7) {
```

Comment:
👾 
Bytes remaining should not be incremented here as this is the amount of bytes we expect for MSP api version 1.40

Reply:
thanks